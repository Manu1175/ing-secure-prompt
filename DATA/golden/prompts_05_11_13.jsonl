{"id": "prompts_05_11_13:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a formal memo summarizing two guidelines: 'Risk Governance Framework' (Issuing Body: National Regulator; Applicability: All Branches; Review Frequency: Bi-Annual; Last Reviewed: 2024-12-04; Status: Deprecated) and 'Cybersecurity Controls' (Issuing Body: Basel Committee; Applicability: HQ Only; Review Frequency: Bi-Annual; Last Reviewed: 2025-04-03; Status: Advisory). Provide next steps for each.", "expected_scrub": "Write a formal memo summarizing two guidelines: '<GUIDELINE_NAME>' (Issuing Body: <ISSUING_BODY>; Applicability: <APPLICABILITY>; Review Frequency: <REVIEW_FREQUENCY>; Last Reviewed: <LAST_REVIEWED_DATE>; Status: <GUIDELINE_STATUS>) and '<GUIDELINE_NAME>' (Issuing Body: <ISSUING_BODY>; Applicability: <APPLICABILITY>; Review Frequency: <REVIEW_FREQUENCY>; Last Reviewed: <LAST_REVIEWED_DATE>; Status: <GUIDELINE_STATUS>). Provide next steps for each.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Post a short Slack reminder: 'AML Screening Guidelines' for Digital Services are currently Status: Deprecated; Review Frequency: Annual; Last Reviewed: 2025-01-29. Ask teams to pause any new control rollouts pending replacement.", "expected_scrub": "Post a short Slack reminder: '<GUIDELINE_NAME>' for <APPLICABILITY> are currently Status: <GUIDELINE_STATUS>; Review Frequency: <REVIEW_FREQUENCY>; Last Reviewed: <LAST_REVIEWED_DATE>. Ask teams to pause any new control rollouts pending replacement.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a risk-acceptance note referencing 'Third-Party Risk Management' (Basel Committee; Applicability: Digital Services; Review Frequency: Bi-Annual; Last Reviewed: 2025-04-09; Status: Deprecated). Propose a temporary control until successor guidance is adopted.", "expected_scrub": "Create a risk-acceptance note referencing '<GUIDELINE_NAME>' (<ISSUING_BODY>; Applicability: <APPLICABILITY>; Review Frequency: <REVIEW_FREQUENCY>; Last Reviewed: <LAST_REVIEWED_DATE>; Status: <GUIDELINE_STATUS>). Propose a temporary control until successor guidance is adopted.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a product launch blurb for 'INGBA4' (Business Account) — Supported Currencies: GBP; Interest Rate: 2.64; Fees: $43 annual fee; Distribution: Online; Regulatory Tags: KYC, FATCA; Operational Status: Active; Version: v2.2; Product Code: BPRL9KDW.", "expected_scrub": "Draft a product launch blurb for '<PRODUCT_NAME>' (<PRODUCT_TYPE>) — Supported Currencies: <SUPPORTED_CURRENCIES>; Interest Rate: <INTEREST_RATE>; Fees: <FEES>; Distribution: <CHANNELS>; Regulatory Tags: <REG_TAGS>; Operational Status: <OP_STATUS>; Version: <VERSION>; Product Code: <PRODUCT_CODE>.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Propose a channel expansion plan for 'INGCC7' (Credit Card) — Supported Currencies: USD, AUD; Interest Rate: 2.20; Fees: $99 annual fee; Distribution: Branch; Regulatory Tags: AML; Operational Status: Active; Version: v1.9; Product Code: BP9VP6DR.", "expected_scrub": "Propose a channel expansion plan for '<PRODUCT_NAME>' (<PRODUCT_TYPE>) — Supported Currencies: <SUPPORTED_CURRENCIES>; Interest Rate: <INTEREST_RATE>; Fees: <FEES>; Distribution: <CHANNELS>; Regulatory Tags: <REG_TAGS>; Operational Status: <OP_STATUS>; Version: <VERSION>; Product Code: <PRODUCT_CODE>.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a short promo caption for 'INGSA7' (Savings Account) — Currencies: JPY; Interest Rate: 2.24; Fees: $35 annual fee; Channels: Online; Regulatory Tags: GDPR, KYC; Status: Active; Version: v3.0; Product Code: BPOXTYWJ.", "expected_scrub": "Create a short promo caption for '<PRODUCT_NAME>' (<PRODUCT_TYPE>) — Currencies: <SUPPORTED_CURRENCIES>; Interest Rate: <INTEREST_RATE>; Fees: <FEES>; Channels: <CHANNELS>; Regulatory Tags: <REG_TAGS>; Status: <OP_STATUS>; Version: <VERSION>; Product Code: <PRODUCT_CODE>.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide a mortgage comparison: 'INGM1' (EUR; 4.27; $64 annual; Mobile, ATM, Branch; FATCA; Active; v2.9; Code BPWY73HD) vs 'INGM4' (JPY, EUR; 1.58; $68 annual; Mobile, Online; KYC, FATCA; Active; v3.1; Code BPNPY80L). Include a one-line recommendation.", "expected_scrub": "Provide a mortgage comparison: '<PRODUCT_NAME>' (...) vs '<PRODUCT_NAME>' (...). Include a one-line recommendation using <SUPPORTED_CURRENCIES>, <INTEREST_RATE>, <FEES>, <CHANNELS>, <REG_TAGS>, <OP_STATUS>, <VERSION>, <PRODUCT_CODE>.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Compile a 'GDPR tag' audit list for two active products: 'INGBA6' (Active; GDPR, FATCA; Mobile, Online; v1.4; Code BP9S6249) and 'INGCC10' (Active; GDPR, AML; Mobile, Online; v2.2; Code BP4DCW3J). Include channels and versions.", "expected_scrub": "Compile a 'GDPR tag' audit list for two active products: '<PRODUCT_NAME_1>' and '<PRODUCT_NAME_2>' with <OP_STATUS>, <REG_TAGS>, <CHANNELS>, <VERSION>, and <PRODUCT_CODE>.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Write acceptance criteria for eligibility checks using a card product 'INGCC4' — Eligibility: 18+ years, valid ID, proof of income; Reg Tags: FATCA, KYC; Status: Active; Version: v1.0; Code: BPS7MXPN.", "expected_scrub": "Write acceptance criteria for eligibility checks using '<PRODUCT_NAME>' — Eligibility: <ELIGIBILITY>; Reg Tags: <REG_TAGS>; Status: <OP_STATUS>; Version: <VERSION>; Code: <PRODUCT_CODE>.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a version-management note for 'INGCC13' — Credit Card, GBP; Interest: 0.84; Fee: $74 annual; Channel: Online; Reg Tag: KYC; Status: Active; Version: v3.9; Code: BP825VJA. Include a rollback reference.", "expected_scrub": "Draft a version-management note for '<PRODUCT_NAME>' — <PRODUCT_TYPE>, <SUPPORTED_CURRENCIES>; Interest: <INTEREST_RATE>; Fee: <FEES>; Channel: <CHANNELS>; Reg Tag: <REG_TAGS>; Status: <OP_STATUS>; Version: <VERSION>; Code: <PRODUCT_CODE>. Include a rollback reference.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a courteous support email to customer Emily Williams confirming her contact details: Email emily.williams@hotmail.com, Phone +32 23 361 947, Street 780 Main St, City Leuven, DOB 1999-11-05 (Age 26). Ask her to confirm if the address is still current.", "expected_scrub": "Write a courteous support email to customer <FIRST_NAME> <LAST_NAME> confirming contact details: Email <EMAIL>, Phone <PHONE>, Street <STREET>, City <CITY>, DOB <DOB> (Age <AGE>). Ask for confirmation if the address is still current.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a fraud-monitoring alert: flag Alex Williams (IP: 167.164.26.227, Email: alex.williams@skynet.be) after multiple device changes; also note Michael Smith (IP: 127.81.31.124, Email: michael.smith@gmail.com) for review due to localhost-range IP usage in logs.", "expected_scrub": "Draft a fraud-monitoring alert: flag <FIRST_NAME> <LAST_NAME> (IP: <IP_ADDRESS>, Email: <EMAIL>) after multiple device changes; also note <FIRST_NAME> <LAST_NAME> (IP: <IP_ADDRESS>, Email: <EMAIL>) for review due to localhost-range IP usage in logs.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Produce a micro-segmentation note for Brussels customers interested in Technology: include Katie Taylor (Email: katie.taylor@telenet.be, City: Brussels) and Tom Taylor (Email: tom.taylor@proximus.be, City: Brussels). Suggest a targeted offer for INGSA7 via Online channel.", "expected_scrub": "Produce a micro-segmentation note for <CITY> customers interested in <INTEREST>: include <FIRST_NAME> <LAST_NAME> (Email: <EMAIL>, City: <CITY>) and <FIRST_NAME> <LAST_NAME> (Email: <EMAIL>, City: <CITY>). Suggest a targeted offer for <PRODUCT_NAME> via <CHANNELS>.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a GDPR data-access confirmation for John Brown (Email: john.brown@telenet.be, City: Brussels) acknowledging receipt of his request and confirming identity checks via phone +32 62 363 308.", "expected_scrub": "Draft a GDPR data-access confirmation for <FIRST_NAME> <LAST_NAME> (Email: <EMAIL>, City: <CITY>) acknowledging receipt of the request and confirming identity checks via phone <PHONE>.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a CRM correction ticket for Laura Miller (Email: laura.miller@telenet.be, City: Liège) to update household data to '1 members' and verify phone +32 55 269 285.", "expected_scrub": "Create a CRM correction ticket for <FIRST_NAME> <LAST_NAME> (Email: <EMAIL>, City: <CITY>) to update household data to '<HOUSEHOLD_DATA>' and verify phone <PHONE>.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a friendly welcome email for Savings Account 'INGSA7' to Emily Taylor (Email: emily.taylor@telenet.be; City: Charleroi) highlighting Interest Rate 2.24, Fee $35 annual, and Online channel.", "expected_scrub": "Write a friendly welcome email for Savings Account '<PRODUCT_NAME>' to <FIRST_NAME> <LAST_NAME> (Email: <EMAIL>; City: <CITY>) highlighting Interest Rate <INTEREST_RATE>, Fee <FEES>, and <CHANNELS> channel.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Prepare a KYC checklist for Alex Johnson (ID: 95.12.08-389.87; Email: alex.johnson@skynet.be; City: Leuven): verify valid ID, proof of income, and age (DOB 1995-12-08, Age 30).", "expected_scrub": "Prepare a KYC checklist for <FIRST_NAME> <LAST_NAME> (ID: <ID_NUMBER>; Email: <EMAIL>; City: <CITY>): verify valid ID, proof of income, and age (DOB <DOB>, Age <AGE>).", "entities": null, "notes": null}
{"id": "prompts_05_11_13:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a gentle contact-update nudge to Tom Jones: Phone +32 85 622 130 appears unreachable; ask him (Email: tom.jones@proximus.be; City: Liège) to confirm preferred number.", "expected_scrub": "Compose a gentle contact-update nudge to <FIRST_NAME> <LAST_NAME>: Phone <PHONE> appears unreachable; ask (Email: <EMAIL>; City: <CITY>) to confirm preferred number.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Send an outage notification to two Ghent customers — Robert Taylor (Email: robert.taylor@proximus.be) and Michael Miller (Email: michael.miller@telenet.be) — informing them of planned maintenance and suggesting Mobile channel for access.", "expected_scrub": "Send an outage notification to two <CITY> customers — <FIRST_NAME> <LAST_NAME> (Email: <EMAIL>) and <FIRST_NAME> <LAST_NAME> (Email: <EMAIL>) — informing them of planned maintenance and suggesting <CHANNELS> for access.", "entities": null, "notes": null}
{"id": "prompts_05_11_13:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_05_11_13.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a data-minimization audit note listing the following data points present in customer records for Michael Smith: Email michael.smith@gmail.com, IP 127.81.31.124, Age 22, City Liège. Recommend masking IPs in analytics.", "expected_scrub": "Create a data-minimization audit note listing the following data points present in customer records for <FIRST_NAME> <LAST_NAME>: Email <EMAIL>, IP <IP_ADDRESS>, Age <AGE>, City <CITY>. Recommend masking IPs in analytics.", "entities": null, "notes": null}
