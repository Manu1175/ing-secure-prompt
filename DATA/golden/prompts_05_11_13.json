[
  {
    "text": "Write a formal memo summarizing two guidelines: 'Risk Governance Framework' (Issuing Body: National Regulator; Applicability: All Branches; Review Frequency: Bi-Annual; Last Reviewed: 2024-12-04; Status: Deprecated) and 'Cybersecurity Controls' (Issuing Body: Basel Committee; Applicability: HQ Only; Review Frequency: Bi-Annual; Last Reviewed: 2025-04-03; Status: Advisory). Provide next steps for each.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Post a short Slack reminder: 'AML Screening Guidelines' for Digital Services are currently Status: Deprecated; Review Frequency: Annual; Last Reviewed: 2025-01-29. Ask teams to pause any new control rollouts pending replacement.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Create a risk-acceptance note referencing 'Third-Party Risk Management' (Basel Committee; Applicability: Digital Services; Review Frequency: Bi-Annual; Last Reviewed: 2025-04-09; Status: Deprecated). Propose a temporary control until successor guidance is adopted.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Draft a product launch blurb for 'INGBA4' (Business Account) — Supported Currencies: GBP; Interest Rate: 2.64; Fees: $43 annual fee; Distribution: Online; Regulatory Tags: KYC, FATCA; Operational Status: Active; Version: v2.2; Product Code: BPRL9KDW.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Propose a channel expansion plan for 'INGCC7' (Credit Card) — Supported Currencies: USD, AUD; Interest Rate: 2.20; Fees: $99 annual fee; Distribution: Branch; Regulatory Tags: AML; Operational Status: Active; Version: v1.9; Product Code: BP9VP6DR.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create a short promo caption for 'INGSA7' (Savings Account) — Currencies: JPY; Interest Rate: 2.24; Fees: $35 annual fee; Channels: Online; Regulatory Tags: GDPR, KYC; Status: Active; Version: v3.0; Product Code: BPOXTYWJ.",
    "labels": [
      "BIC",
      "PHONE"
    ]
  },
  {
    "text": "Provide a mortgage comparison: 'INGM1' (EUR; 4.27; $64 annual; Mobile, ATM, Branch; FATCA; Active; v2.9; Code BPWY73HD) vs 'INGM4' (JPY, EUR; 1.58; $68 annual; Mobile, Online; KYC, FATCA; Active; v3.1; Code BPNPY80L). Include a one-line recommendation.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Compile a 'GDPR tag' audit list for two active products: 'INGBA6' (Active; GDPR, FATCA; Mobile, Online; v1.4; Code BP9S6249) and 'INGCC10' (Active; GDPR, AML; Mobile, Online; v2.2; Code BP4DCW3J). Include channels and versions.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Write acceptance criteria for eligibility checks using a card product 'INGCC4' — Eligibility: 18+ years, valid ID, proof of income; Reg Tags: FATCA, KYC; Status: Active; Version: v1.0; Code: BPS7MXPN.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft a version-management note for 'INGCC13' — Credit Card, GBP; Interest: 0.84; Fee: $74 annual; Channel: Online; Reg Tag: KYC; Status: Active; Version: v3.9; Code: BP825VJA. Include a rollback reference.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Write a courteous support email to customer Emily Williams confirming her contact details: Email emily.williams@hotmail.com, Phone +32 23 361 947, Street 780 Main St, City Leuven, DOB 1999-11-05 (Age 26). Ask her to confirm if the address is still current.",
    "labels": [
      "DOB",
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Draft a fraud-monitoring alert: flag Alex Williams (IP: 167.164.26.227, Email: alex.williams@skynet.be) after multiple device changes; also note Michael Smith (IP: 127.81.31.124, Email: michael.smith@gmail.com) for review due to localhost-range IP usage in logs.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Produce a micro-segmentation note for Brussels customers interested in Technology: include Katie Taylor (Email: katie.taylor@telenet.be, City: Brussels) and Tom Taylor (Email: tom.taylor@proximus.be, City: Brussels). Suggest a targeted offer for INGSA7 via Online channel.",
    "labels": [
      "EMAIL"
    ]
  },
  {
    "text": "Draft a GDPR data-access confirmation for John Brown (Email: john.brown@telenet.be, City: Brussels) acknowledging receipt of his request and confirming identity checks via phone +32 62 363 308.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Create a CRM correction ticket for Laura Miller (Email: laura.miller@telenet.be, City: Liège) to update household data to '1 members' and verify phone +32 55 269 285.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Write a friendly welcome email for Savings Account 'INGSA7' to Emily Taylor (Email: emily.taylor@telenet.be; City: Charleroi) highlighting Interest Rate 2.24, Fee $35 annual, and Online channel.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Prepare a KYC checklist for Alex Johnson (ID: 95.12.08-389.87; Email: alex.johnson@skynet.be; City: Leuven): verify valid ID, proof of income, and age (DOB 1995-12-08, Age 30).",
    "labels": [
      "DOB",
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Compose a gentle contact-update nudge to Tom Jones: Phone +32 85 622 130 appears unreachable; ask him (Email: tom.jones@proximus.be; City: Liège) to confirm preferred number.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Send an outage notification to two Ghent customers — Robert Taylor (Email: robert.taylor@proximus.be) and Michael Miller (Email: michael.miller@telenet.be) — informing them of planned maintenance and suggesting Mobile channel for access.",
    "labels": [
      "EMAIL"
    ]
  },
  {
    "text": "Create a data-minimization audit note listing the following data points present in customer records for Michael Smith: Email michael.smith@gmail.com, IP 127.81.31.124, Age 22, City Liège. Recommend masking IPs in analytics.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  }
]